src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
+    public static void printList(List<Task> list) {
+        for (int i = 0; i < list.size(); i++) {
+            System.out.println((i+1)+"."+list.get(i).getDoneStatus()+" "+list.get(i).getDescription());
+        }
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String lineBreak = "----------------------------------------------------------";
+        String idle = "       █████████████    \n"
+                    + "      ██           ██   \n"
+                    + "      ██   █████   ██   \n"
+                    + "      ██   █████   ██   \n"
+                    + "      ██           ██   \n"
+                    + "       █████████████    \n"
+                    + "   ██                  ██\n"
+                    + "     ██████████████████ \n\n";
+
+        String happy= "            ███         \n"
+                    + "          ███████       \n"
+                    + "         ████ ████      \n"
+                    + "        ███     ███     \n"
+                    + "       ██         ██    \n"
+                    + "   ██                 ██\n"
+                    + "    ███             ███ \n"
+                    + "      ███████████████   \n\n";
+
+        String input = "Start";
+        List<Task> list = new ArrayList<>(2);
+        Scanner in = new Scanner(System.in);
+        System.out.println(idle + "Hello, Im Pythia, how may I help you today?\n"+lineBreak);
+        while (!input.equals("bye")) {
+            input = in.nextLine();
+            if (input.equalsIgnoreCase("list")) {
+                printList(list);
+            }
+            else if (input.contains("add ")) {
```

Comment:
regarding the else if statement, the coding standard should be something like this 
if (....) {
     xxx;
} else if {
     yyyy;
} else {
     zzzz;
}